src/GafferSceneUIModule/SetEditorBinding.cpp

```diff
@@ -278,6 +280,20 @@ class SetPath : public Gaffer::Path
 					return new IntData( setMembers->readable().size() );
 				}
 			}
+			else if( name == g_selectedMemberCountPropertyName )
+			{
+				const PathMatcher p = pathMatcher( canceller );
+				if( p.match( names() ) & PathMatcher::ExactMatch )
+				{
+					Context::EditableScope scopedContext( getContext() );
+					if( canceller )
+					{
+						scopedContext.setCanceller( canceller );
+					}
+					const auto setMembers = getScene()->set( names().back().string() );
+					return new IntData( setMembers->readable().intersection( ContextAlgo::getSelectedPaths( Context::current() ) ).size() );
```

Comment:
This is the equivalent of `set & selectionSet`, and will only consider selected locations that are explicitly included in the set. I wonder if we actually want to do the equivalent of `selectionSet in set` so that we consider membership through inheritance? Or do we need to show both somehow?

Reply:
The other thing we might want to consider is a way of getting the size of an intersection (or `in`) without actually having to generate a new PathMatcher - I think that could be a fair bit quicker.

Reply:
I've switched over to counting selected locations via inheritance in 2e16a6e & 444d837. This does seem more useful/informative than the previous approach, and hopefully not too confusing in presentation. On that note, and based on the suggestion below, I've added some new columns for the display of "Members" and "Selection" in 8a4099a in order to introduce cell and header tooltips describing the behaviour.